chain-test-utils/src/generators/utils/constant_generator.rs

```diff
@@ -0,0 +1,18 @@
+use crate::generators::utils::{Generator, R};
+
+#[derive(Clone)]
+pub struct ConstantGenerator<T>(T);
```

Comment:
For some reason, the one in `smoke` wasn't public.

Reply:
make it public, push a PR

Reply:
My bad, actually it was public but starting lowercase 😅